RecoEcal/EgammaClusterProducers/python/particleFlowSuperClusterECAL_cfi.py

```diff
@@ -154,3 +154,8 @@
 
 #define the default clustering type
 particleFlowSuperClusterECAL = particleFlowSuperClusterECALMustache.clone()
+
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+pp_on_AA_2018.toModify(particleFlowSuperClusterECAL, useDynamicDPhiWindow = False
```

Comment:
where are the closing braces?

Was this code checked?
http://cms-sw.github.io/PRWorkflow

Reply:
sorry, this was fixed and tested locally but the changes were not committed.